cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Returns the Tags that are often mis-labeled.
+
+
+    This method provides two scores in the Pandas DataFrame that is returned:
+
+    * **num_examples**: The number of examples where the given/predicted label differ.
+    * **Issue_probability**: `(num_examples / total number of examples in the dataset`).
+
+    This method works by providing any one (and only one) of the following inputs:
+
+    1. ``labels`` and ``pred_probs``, or
+    2. ``confident_joint``
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+        Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
+
+    pred_probs : np.ndarray, optional
+      Refer to documentation for this argument in filter._find_multilabel_issues_per_class() for further details.
```

Comment:
you are not properly linking the method. Look at the DEVELOPMENT.md guidelines for instructions on how to properly link to the other method in a docstring. Also you still should give a one-line description of the argument before you link to it (just copy over the first line of the description from the other function).

Reply:
Fixed here: https://github.com/cleanlab/cleanlab/blob/16cffafd6b397cfb7edf567a697aa537230b4ac6/cleanlab/multilabel_classification/dataset.py#L36